### ISSUE TITLE
LostAndFoundDataset

### ISSUE THREAD

**Issue by RamyaRaghuraman** [cvlab-epfl/detecting-the-unexpected#5]
Hi @adynathos 

For the LostAndFoundDataset from http://www.6d-vision.com/lostandfounddataset, what are the packages that we should download?

Also is the downscaled version of cityscapes available?
DIR_CITYSCAPES = Path(os.environ.get('DIR_CITYSCAPES', DIR_DSETS / 'dataset_Cityscapes' / '2048x1024'))
DIR_CITYSCAPES_SMALL = Path(os.environ.get('DIR_CITYSCAPES_SMALL', DIR_DSETS / 'dataset_Cityscapes' / '1024x512'))

**Comment by adynathos** [comment]
Hello,
We download the images (leftImg8bit) and labels (gtCoarse), then resize them with a script.
[Please see the description here](https://github.com/cvlab-epfl/detecting-the-unexpected#dataset-processing)

**Comment by RamyaRaghuraman** [comment]
I end up with the following error:

 File "compress_images.py", line 114, in <module>
    main()
  File "C:\Users\RAR7ABT\AppData\Local\conda\conda\envs\test\lib\site-packages\click\core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "C:\Users\RAR7ABT\AppData\Local\conda\conda\envs\test\lib\site-packages\click\core.py", line 717, in main
    rv = self.invoke(ctx)
  File "C:\Users\RAR7ABT\AppData\Local\conda\conda\envs\test\lib\site-packages\click\core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "C:\Users\RAR7ABT\AppData\Local\conda\conda\envs\test\lib\site-packages\click\core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "compress_images.py", line 110, in main
    async_map(run_external_program, cmds, num_concurrent=concurrent)
  File "compress_images.py", line 35, in async_map
    asyncio.get_event_loop().run_until_complete(joint_future)
  File "C:\Users\RAR7ABT\AppData\Local\conda\conda\envs\test\lib\asyncio\base_events.py", line 484, in run_until_complete
    return future.result()
  File "compress_images.py", line 25, in async_worker
    result = await func(task)
  File "compress_images.py", line 56, in run_external_program
    raise e
  File "compress_images.py", line 50, in run_external_program
    stderr=asyncio.subprocess.PIPE,
  File "C:\Users\RAR7ABT\AppData\Local\conda\conda\envs\test\lib\asyncio\subprocess.py", line 225, in create_subprocess_exec
    stderr=stderr, **kwds)
  File "C:\Users\RAR7ABT\AppData\Local\conda\conda\envs\test\lib\asyncio\base_events.py", line 1211, in subprocess_exec
    bufsize, **kwargs)
  File "C:\Users\RAR7ABT\AppData\Local\conda\conda\envs\test\lib\asyncio\coroutines.py", line 212, in coro
    res = func(*args, **kw)
  File "C:\Users\RAR7ABT\AppData\Local\conda\conda\envs\test\lib\asyncio\base_events.py", line 360, in _make_subprocess_transport
    raise NotImplementedError
NotImplementedError

I do have Libwebp and Imagemagick installed.

**Comment by RamyaRaghuraman** [comment]
Hi @adynathos ,

I was able to overcome the previous error by adding the following in compress_images.py:

import asyncio, sys
if sys.platform == 'win32':
    loop = asyncio.ProactorEventLoop()
    asyncio.set_event_loop(loop)

However i was able to downsample only the leftImg8bit images and not the gtCoarse images. I got the following error for all the images:

--Stderr--
b'Invalid Parameter - -filter\r\n'
--Stdout--
b''

**Comment by RamyaRaghuraman** [comment]
Replacing "convert" to "magick" (line 17) in the sh files fixes the issue.

**Comment by adynathos** [comment]
Thank you for finding that. I did not run the software on Windows and was not aware of the problem.
I added your fixes here 4035478c3548563e1c4ca38e716b23b33d5daa6e

**Comment by a9311072** [comment]
for running the resize
```
sudo apt-get update
sudo apt-get install webp
sudo apt-get install imagemagick
```
